.github/workflows/run-standard-tests.yml

```diff
@@ -0,0 +1,194 @@
+          if [[ "${{ matrix.running-mode }}" == "parallel" ]] ; then 
+            export KhiopsMPIProcessNumber=3 
+          fi
+          if [[ "${{ matrix.config }}" == "release" ]] ; then
+            python $TEST_PY Khiops ${BIN_PATH}/MODL${EXT} Standard
+            if [[ "${{ matrix.running-mode }}" != "parallel" ]] ; then
+              python $TEST_PY Coclustering ${BIN_PATH}/MODL_Coclustering${EXT} Standard
+              python $TEST_PY KNI ${BIN_PATH}/KNITransfer${EXT} Standard
+              python $TEST_PY KNI ${BIN_PATH}/KNITransfer${EXT} MultiTables SpliceJunction
+            fi
+          else
+            python $TEST_PY Khiops ${BIN_PATH}/MODL${EXT} Standard IrisLight
+            if [[ "${{ matrix.running-mode }}" != "parallel" ]] ; then
+              python $TEST_PY Coclustering ${BIN_PATH}/MODL_Coclustering${EXT} Standard Iris
+              python $TEST_PY KNI ${BIN_PATH}/KNITransfer${EXT} Standard Iris
+            fi
+          fi
+      - name: Collect results
+        shell: bash
+        if: success() || failure()
+        run: |
+          if [[ "${{ matrix.config }}" == "release" ]] ; then
+            python3 $APPLY_PY errors test/LearningTest/TestKhiops/Standard | tee test/LearningTest/TestKhiops/Standard/errors.txt
+            if [[ "${{ matrix.running-mode }}" != "parallel" ]] ; then
+              python $APPLY_PY errors test/LearningTest/TestCoclustering/Standard | tee test/LearningTest/TestCoclustering/Standard/errors.txt
+              python $APPLY_PY errors test/LearningTest/TestKNITransfer/Standard | tee test/LearningTest/TestKNITransfer/Standard/errors.txt
+              python $APPLY_PY errors test/LearningTest/TestKNITransfer/MultiTables SpliceJunction | tee test/LearningTest/TestKNITransfer/MultiTables/errors.txt
+            fi
+          else
+            python3 $APPLY_PY errors test/LearningTest/TestKhiops/Standard IrisLight | tee test/LearningTest/TestKhiops/Standard/errors.txt
+            if [[ "${{ matrix.running-mode }}" != "parallel" ]] ; then
+              python $APPLY_PY errors test/LearningTest/TestCoclustering/Standard Iris | tee test/LearningTest/TestCoclustering/Standard/errors.txt
+              python $APPLY_PY errors test/LearningTest/TestKNITransfer/Standard Iris | tee test/LearningTest/TestKNITransfer/Standard/errors.txt
+            fi
+          fi
+      - name: Check results
+        shell: bash
+        run: |
+          if grep -qr "error" --include="errors.txt" test/LearningTest/ ; then
+            echo "::error::Errors in test"
+            false
+          fi
+          if grep -qr "The test has not been launched" --include="errors.txt" test/LearningTest/ ; then
+            echo "::error::Test not launched"
+            false
+          fi
+      - name: Upload test results
+        if: failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.build-setup.cmake-preset }}-${{ matrix.config }}-${{ matrix.running-mode
```

Comment:
Rename to ``test-results-${{ env.PRESET_NAME }}``

Reply:
OK renamed to test-results-${{ env.PRESET_NAME }}**-${{ matrix.running-mode}}**

Reply:
Ok